packages/core/deploy/002_deploy_timer.js

```diff
@@ -0,0 +1,20 @@
+const func = async function (hre) {
+  const { deployments, getNamedAccounts, network } = hre;
+  const { deploy } = deployments;
+
+  const { deployer } = await getNamedAccounts();
+  const { live } = network;
+
+  if (live === undefined) throw new Error("Network has no live parameter");
```

Comment:
Leaving this here in case some configuration change in hardhat causes this to be undefined. We would want to catch that to avoid accidentally deploying a Timer in prod.

Reply:
I agree with leaving this here